File: BAEKJOON/Javascript/5000/5300/5355.js
const fs = require("fs");
const filepath =
  process.platform === "linux" ? "/dev/stdin" : __dirname + "/input.txt";
const input = fs.readFileSync(filepath).toString().trim().split("\n");

const t = parseInt(input[0]);
for (let i = 1; i <= t; i++) {
  const line = input[i].trim().split(" ");
  let n = parseFloat(line[0]);
  for (let j = 1; j < line.length; j++) {
    let cmd = line[j];
    if (cmd === "@") {
      n *= 3;
    } else if (cmd === "%") {
      n += 5;
    } else if (cmd === "#") {
      n -= 7;
    }
  }
  console.log(n.toFixed(2));
}

File: BAEKJOON/Golang/5000/5300/5355.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	sc := bufio.NewScanner(os.Stdin)

	sc.Scan()
	t, _ := strconv.Atoi(sc.Text())
	for i := 0; i < t; i++ {
		sc.Scan()
		line := strings.Split(sc.Text(), " ")
		n, _ := strconv.ParseFloat(line[0], 32)
		for _, cmd := range line {
			if cmd == "@" {
				n *= 3
			} else if cmd == "%" {
				n += 5
			} else if cmd == "#" {
				n -= 7
			}
		}
		fmt.Printf("%.2f\n", n)
	}
}
